src/checkers/human_checker.rs

```diff
@@ -1,15 +1,18 @@
 use crate::checkers::checker_result::CheckResult;
-
-#[cfg(not(test))]
+use crate::config::CONFIG;
 use text_io::read;
 
 /// The Human Checker asks humans if the expected plaintext is real plaintext
 /// We can use all the automated checkers in the world, but sometimes they get false positives
 /// Humans have the last say.
 /// TODO: Add a way to specify a list of checkers to use in the library. This checker is not library friendly!
 // compile this if we are not running tests
-#[cfg(not(test))]
 pub fn human_checker(input: &CheckResult) -> bool {
+    let config = CONFIG.wait(); // wait instead of get so it waits for config being set
+    if !config.human_checker_on {
```

Comment:
```suggestion
    // We still call human checker, just if config is false we return True
    if !config.human_checker_on {
```

Reply:
How does this work with the timer?

Reply:
I don't think it'll affect it much, if timer is on and athena returns true we just record the record